scrapers/base.py

```diff
@@ -62,14 +62,12 @@ def is_container_build(self, build_info):
         except (ValueError, TypeError):
             extra_json = {}
 
-        # Checking a heuristic for determining if a build is a container build since, currently
+        # Checking heuristics for determining if a build is a container build since, currently
```

Comment:
The comma in the comment is actually in the wrong place. I wouldn't have said anything except you're modifying that line :stuck_out_tongue:

Reply:
hahaha, i'll fix it